tests/modules/commandfilter.c

```diff
@@ -89,6 +89,29 @@ int CommandFilter_LogCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int
     return REDISMODULE_OK;
 }
 
+static int brpoplpush_count = 0;
+
+/* Filter to protect against Bug #11894 reappearing */
+void CommandFilter_BlpopFilter(RedisModuleCommandFilterCtx *filter)
+{
+    if (RedisModule_CommandFilterArgsCount(filter) != 3)
+        return;
+
+    const RedisModuleString *arg = RedisModule_CommandFilterArgGet(filter, 0);
+    size_t arg_len;
+    const char *arg_str = RedisModule_StringPtrLen(arg, &arg_len);
+
+    if (arg_len != 10 || memcmp(arg_str, "brpoplpush", 10))
```

Comment:
maybe a simple strncmp would be clearer.

Reply:
changed